app/views/spree/shared/_sign_in_header.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav class="breadcrumb-wrap d-flex flex-wrap align-items-center justify-content-between">
+   <ol class="breadcrumb">
+      <li class="breadcrumb-item"><a href="#">Home</a></li>
+      <li class="breadcrumb-item active">Sign In</li>
+   </ol>
+</nav>
+<h1 class="title">Sign In</h1>
```

Comment:
Remove this partial and move the bewlo 2 line into devise -> User signin file.
<h1 class="title">Sign In</h1>
<div class="signin-note">Welcome to our new site. If you had an account on our previous system, please reset your password to access the new system by click <%= link_to Spree.t(:forgot_your_password?), spree.recover_password_path %>. Contact us if you need further assistance.</div>